packages/portfolio-contract/src/portfolio.exo.ts

```diff
@@ -192,6 +192,7 @@ export const preparePortfolioKit = (
       positionId,
       accountId: coerceAccountId(address),
       ...emptyTransferState,
+      usdcBrand: undefined as Brand<'nat'> | undefined,
```

Comment:
Using an amount from `want` means this isn't necessary.

(Also: the brand is available at construction time; no need for `| undefined`.)